src/pfe/file-watcher/server/test/functional-test/lib/utils.ts

```diff
@@ -164,3 +164,27 @@ export async function rebuildProject(socket: SocketIO, projData: ProjectCreation
         fail(`failed to find ${targetEvent} for rebuild project ${projData.projectType}`);
     }
 }
+
+export async function removeProjectFromRunningBuild(projData: ProjectCreation): Promise<void> {
```

Comment:
probably add a comment here like above why we're doing this specifically

Reply:
Done.